settings.gradle.kts

```diff
@@ -170,3 +172,14 @@ pluginManagement {
     }
 }
 
+val ignoredFeatures = setOf(
+    // we don't want to publish Gradle metadata to public repositories until the format is stable.
+    FeaturePreviews.Feature.GRADLE_METADATA
+)
+
+FeaturePreviews.Feature.values().forEach { feature ->
+    if (feature.isActive && !ignoredFeatures.contains(feature)) {
+        enableFeaturePreview(feature.name)
+    }
+}
```

Comment:
Why not default in the production code to _enabled,_ and use the production feature (=a switch in `gradle.properties`) to disable `GRADLE_METADATA`?

Reply:
What do you mean by "a switch in `gradle.properties`"? I am not aware that you can turn on/off features by using `gradle.properties`.

By production code, do you mean `settings.gradle.kts`?

Reply:
No, by production code I mean changing `FeaturePreviews` to consider features enabled by default, and to allow disabling them.

I was quite sure that we had a way to enable features via properties. Why wouldn't we allow that? :o

Reply:
You're right. I think I need some sleep, sorry. :)